### DISCUSSION TITLE
How about separating jobs and job runner into different module not part of the server?

### DISCUSSION THREAD

**Discussion by suhaibaffan** [bigcapitalhq/bigcapital#131] [Q&A] [answered]
For instance if I have deployed the api server on Kubernetes with multiple pods, this will lead to run jobs on every pod.
This could lead in triggering jobs multiple times depending on the number of replicas.

**Comment by abouolia** [comment, marked as answer]
Until now the application still stateful, if you want to run it on Kubernetes must be distributed stateless application for scalability and that what we're seeking to in the next versions.

**Comment by suhaibaffan** [comment]
Thank you @abouolia can you create an issue for this? I would like to separate Job runner from the api server.
